Update Docker Enterprise Edition documentation (#1815)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/docker-enterprise-edition.md

1. Follow the instructions to [Collect Logs and Metrics for Docker ULM](/docs/integrations/containers-orchestration/docker-ulm#collecting-logs-and-metrics-for-docker-ulm). Use the same `_sourceCategory` for each host on your cluster.
1. Follow the instructions to [Install the Docker ULM App](/docs/integrations/containers-orchestration/docker-ulm#installing-the-docker-ulm-app) and dashboards.

### Step 2. Deploy Sumo Logic Collection Helm Chart to collect logs and metrics

> For details please see [Kubernetes Observability](/docs/observability/kubernetes) and check [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart.md) to see installation instruction.

Reviewer: ```suggestion
For details see [Kubernetes Observability](/docs/observability/kubernetes) and check the [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart.md) for installation instructions.
```
